mautic/mautic
# feat: DPMMA-1572 Clone campaign events

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ Y ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds possibility to clone (copy/paste) campaign events. 
Cloning events between campaigns is supported with this feature.

![image](https://github.com/mautic/mautic/assets/8580942/86617085-39a0-4e4f-be76-f4d47272181d)
![image](https://github.com/mautic/mautic/assets/8580942/b1492b07-06df-4909-87ab-ac5b20535d0d)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a campaign with some events
3. Clone some events and paste them into other branches or other campaigns
4. Run the campaign and check that it works properly
